frontend/src/types/CropTypesDatabase.ts

```diff
@@ -0,0 +1,8 @@
+export default interface CropTypesDatabase {
```

Comment:
Also here. It's not 100% needed, but it feels silly to have tiny files like this.